### PR TITLE
WT-12128 Make checksum logging more consistent

### PR DESCRIPTION
**Pull request by y123456yz**
image
One output in decimal and one in hexadecimal, keeping the output format consistent and helping us to understand the code.

### CODE REVIEW

**Comment by y123456yz** [comment]
WT-12128

File: src/block/block_addr.c [hunk 1]
```diff
@@ -354,28 +354,28 @@ __wt_ckpt_verbose(WT_SESSION_IMPL *session, WT_BLOCK *block, const char *tag, co
         WT_ERR(__wt_buf_catfmt(session, tmp, ", root=[Empty]"));
     else
         WT_ERR(__wt_buf_catfmt(session, tmp,
-          ", root=[%" PRIuMAX "-%" PRIuMAX ", %" PRIu32 ", %" PRIu32 "]",
+          ", root=[off: %" PRIuMAX "-%" PRIuMAX ", size: %" PRIu32 ", checksum: 0x%08x]",
```
Comment (ershov) [inline src/block/block_addr.c:357]: Please use `PRIx32` and/or friends here and in other places.

**Review by ershov — CHANGES_REQUESTED** [review]
Hi @y123456yz, thank you for your PR!

File: src/block/block_addr.c (hunk 1 above)
Comment (y123456yz) [reply src/block/block_addr.c]: hi, @ershov 
Thank you for your guidance，I have change the code, please review again.

**Review by ershov — APPROVED** [review]
Looks good, thanks @y123456yz !

**Comment by ershov** [comment]
evergreen merge

**Pull request merged**